ZenPacks/zenoss/WSMAN/datasources/WSMANDataSource.py

```diff
@@ -257,10 +257,10 @@ def onSuccess(self, results, config):
                 timestamp = 'N'
 
             for datapoint in datasource.points:
-                if hasattr(result, datapoint.id):
+                value = getattr(result, datapoint.id, None)
+                if value:
```

Comment:
This should if `if value is not None:`. You're preventing the value `0` from being stored with just `if value:`.